tests/app/service/test_rest.py

```diff
@@ -1217,41 +1195,44 @@ def test_set_sms_prefixing_for_service_cant_be_none(
     assert resp['message'] == {'prefix_sms': ['Field may not be null.']}
 
 
-# This always returns 0. Does not appear to be used
-@pytest.mark.skip(reason='Mislabelled for route removal, fails when unskipped')
+@pytest.mark.skip(reason='TODO #2335 - Expected behavior unknown')
+@pytest.mark.serial
 @pytest.mark.parametrize(
     'today_only,stats',
     [('False', {'requested': 2, 'delivered': 1, 'failed': 0}), ('True', {'requested': 1, 'delivered': 0, 'failed': 0})],
     ids=['seven_days', 'today'],
 )
-def test_get_detailed_service(notify_api, notify_db_session, sample_service, sample_template, today_only, stats):
-    with notify_api.test_request_context(), notify_api.test_client() as client:
-        service = sample_service()
-        template = sample_template()
-        ft_notification = create_ft_notification_status(date(2000, 1, 2), 'sms', template=template, count=1)
-        with freeze_time('2000-01-02T12:00:00'):
-            notification_0 = create_notification(template=template, status='created')
-            notification_1 = create_notification(template=template, status='created')
-            resp = client.get(
-                '/service/{}?detailed=True&today_only={}'.format(service.id, today_only),
-                headers=[create_admin_authorization_header()],
-            )
-
-    assert resp.status_code == 200
-    service_resp = resp.json['data']
-
-    assert service_resp['id'] == str(service.id)
-    assert 'statistics' in service_resp
-    assert set(service_resp['statistics'].keys()) == {SMS_TYPE, EMAIL_TYPE, LETTER_TYPE}
-    assert service_resp['statistics'][SMS_TYPE] == stats
-
-    # Teardown
-    notify_db_session.session.delete(notification_0)
-    notify_db_session.session.delete(notification_1)
-    notify_db_session.session.delete(ft_notification)
-    notify_db_session.session.commit()
-
-
+def test_get_detailed_service(
+    notify_api, notify_db_session, sample_service, sample_template, sample_notification, today_only, stats
+):
+    try:
```

Comment:
Others may say the try could be lower, around asserts but this does need to be near the top, prior to the `create_ft_notification_status` call.

No action required, just pointing this out for others.